morpheus/models/dfencoder/autoencoder.py

```diff
@@ -905,11 +905,11 @@ def _fit_centralized(self, df, epochs=1, val=None, run_validation=False, use_val
 
                     # Early stopping
                     current_net_loss = net_loss
-                    LOG.debug('The Current Net Loss:', current_net_loss)
+                    LOG.debug('The Current Net Loss: {}'.format(current_net_loss))
```

Comment:
Use lazy formatting for logger messages:
```suggestion
                    LOG.debug('The Current Net Loss: %s', current_net_loss)
```

Reply:
@mdemoret-nv waiting on customer feedback before closing #816.